### ISSUE TITLE
inline frame missing a text alternative [captcha]

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#3945]
Under WCAG 2.2 CAPTCHA should have a text description of what they are for, and an accessible alternative.

**Comment by StephDriver** [comment]
This will be fixed by #4058

**Comment by StephDriver** [comment]
related work is happening, this will be closed when that is.

**Comment by joemull** [comment]
See https://github.com/BirkbeckCTP/bau/issues/39

**Comment by joemull** [comment]
Now that testing of hcaptcha rollout in https://github.com/BirkbeckCTP/janeway/issues/4058 is complete, this can be closed.
